app/replica.go

```diff
@@ -233,19 +234,25 @@ func startReplica(c *cli.Context) error {
 		status := s.Replica().GetCloneStatus()
 		if status != "completed" {
 			logrus.Infof("Set clone status as inProgress")
-			s.Replica().SetCloneStatus("inProgress")
-			if err = CloneReplica(s, "tcp://"+address, cloneIP, snapName); err != nil {
-				logrus.Infof("Set clone status as error")
-				s.Replica().SetCloneStatus("error")
+			if err := s.Replica().SetCloneStatus("inProgress"); err != nil {
 				return err
 			}
+			if err = CloneReplica(s, "tcp://"+address, cloneIP, snapName); err != nil {
+				logrus.Error("Error in cloning replica, found error ", err.Error())
+				logrus.Info("Setting the status of clone as error")
```

Comment:
why we need this after having Error log?

Reply:
We are adding this info to notify that we are setting the clone status as error. Should we remove this ?